Makefile

```
@@ -1,67 +1,165 @@
 	@echo ""
-	@awk 'BEGIN {FS = ":.*##"} /^[a-zA-Z_-]+:.*?##/ { printf "  ${YELLOW}%-15s${NC} %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
+	@awk 'BEGIN {FS = ":.*##"} /^[a-zA-Z_-]+:.*?##/ { printf "  ${YELLOW}%-20s${NC} %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
 	@echo ""
 
-build: ## Compile and install ktn-linter
-	@echo "${GREEN}Compiling ktn-linter ${VERSION}...${NC}"
+# ---------------------------------------------------------------------------
+# Build
+# ---------------------------------------------------------------------------
+
+build: ## Compile ktn-linter via Bazel and copy to ./builds/ and ~/.local/bin
```

checkmake (0.3.2)

[warning] Target body for "build" exceeds allowed length of 5 lines (8).

(maxbodylength)